getting-started/.gitignore

```diff
@@ -0,0 +1,21 @@
+hotspot.log
```

Comment:
I would suggest moving all this to the root `.gitignore`, but since this a sample app, it should be copyable into another directory and good-to-go from there.

We should take some of these, such as `target/`, and put them in the root `.gitignore` as well. I can do this in a separate PR